packages/experimental/RadioGroup/src/Radio/Radio.types.ts

```diff
@@ -38,6 +38,11 @@ export interface RadioTokens extends FontTokens, IColorTokens, IForegroundColorT
    */
   radioBorderWidth?: number;
 
+  /**
+   * Font size of label subtext
+   */
+  subtextSize?: number;
```

Comment:
Should this be a variant instead of a direct size number? I recognize that the example just used size, but in general we should be steering people towards using font alias tokens/variants over global tokens/size constants